.docker/Dockerfile.rhel

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/rhscl/nodejs-8-rhel7
+FROM registry.access.redhat.com/ubi8/nodejs-12
```

Comment:
Eh.. why this instead of rhscl?

Reply:
Ever since [ubi was announced last year](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image), we've been advocating for customers and partners to use *it* as a base in favor of `rhscl` because `ubi` images can be freely redistributed without requiring RHEL subscriptions, whereas `rhscl` ones can't.

Reply:
Ah!  I missed that announcement!